config-templates/redhat/odf-remote/4.7/custom-parameters.json

```diff
@@ -97,6 +114,10 @@
     "displayname": "Encryption enabled",
     "name": "cluster-encryption",
     "default": "false",
+    "regex": "^(?:tru|fals)e$",
```

Comment:
same here `"regex": "^(?:true|false)$",`

Reply:
Addressed in https://github.com/IBM/ibm-satellite-storage/pull/194/commits/3638379ea315df3f1df332997fb75a5000b13224